Doc/library/sqlite3.rst

```diff
@@ -640,6 +640,11 @@ Cursor Objects
       .. versionchanged:: 3.6
          Added support for the ``REPLACE`` statement.
 
+   .. attribute:: arraysize
+
+      This attribute allows you to set the number of rows returned by :meth:`fetchmany`.
+      The default value is 1 meaning a single row would be fetched per call.
```

Comment:
Even though it is used in the preceding attribute description, I think we usually avoid referring to "you".  So something more like "This attribute controls the number of rows...".  I think there should be a comma after '1' in the second sentence.  It also might read better if it was 'which means...will be' instead of 'meaning', but that's debatable :)